Measurements/Tests.qs

```diff
@@ -378,6 +389,22 @@ namespace Quantum.Kata.Measurements {
 
         CheckSuperpositionBitstringsMeasurement(5, [30,14,10,7],  // [11110,01110,01010,00111]
                                                    [1,17,21,27]); // [00001,10001,10101,11011]
+
+        CheckSuperpositionBitstringsMeasurement(2, [2,1],       // [10,01]
```

Comment:
Please make sure that the input vectors match the constraints described in `Tasks.qs`. For this task, `The only constraint on the bit strings is that all bit strings in the two arrays are distinct. `, so all numbers in the arrays have to be distinct, you can't have element 1 in both arrays.

Reply:
done